src/components/Course/Content/ExerciseContent/index.js

```diff
@@ -25,7 +25,8 @@ ${code}
 const createVisulizeURL = (code, lang, mode) => {
   // only support two languages for now
   let l = lang == "python" ? "2" : "js";
-  let visualizerCode = code && code.replace(/<br>/g, "\n");
+  let replacedCode = code && code.replace(/<br>/g, "\n");
+  let visualizerCode = replacedCode.replace(/&emsp;/g, "    ");
```

Comment:
@Poonam-Singh-Bagh How many spaces for one `&emsp;` ?

Reply:
4

Reply:
No. We discussed one space for one `&emsp`.

Reply:
But it's not looking fine